products/idn/docs/identity-now/rules/guides/your_first_rule.md

```diff
@@ -0,0 +1,495 @@
+---
+id: your-first-rule
+title: Your First Rule
+pagination_label: Your First Rule
+sidebar_label: Your First Rule
+sidebar_class_name: yourFirstRule
+keywords: ['rules', 'guides', 'first']
+description: Learn to build your first rule!
+sidebar_position: 1
+slug: /docs/rules/guides/your-first-rule
+tags: ['Rules', 'Guides', 'First']
+---
+
+## Overview
+
+In this guide you'll learn the end to end process of writing a cloud rule to generate a username to be used during account creation.
+
+- [Attribute generator rule overview](#attribute-generator-rule)
+- [Username requirements](#username-requirements)
+- [Writing the rule](#writing-the-rule)
+- [Validating the rule](#validate-the-rule)
+- [Submitting for rule review](#submitting-for-rule-review)
+
+## Attribute Generator rule
```

Comment:
Please use title case for all section headers.  In this instance, "rule" needs to be capitalized.  Please review all other section headers for title case.